assembl/static2/js/app/components/common/tree/treeItem.jsx

```diff
@@ -55,7 +55,7 @@ class Child extends React.PureComponent<Props, State> {
   };
 
   state = {
-    expanded: true,
+    expanded: false,
```

Comment:
Oh, global for every screen size?

Reply:
From a UX perspective, it should not be expanded since the message box is so huge... but well let's rollback that one (I remember that was a request from Frank for BM).